files/zh-cn/web/api/sourcebuffer/remove/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: SourceBuffer.remove()
+slug: Web/API/SourceBuffer/remove
+page-type: web-api-instance-method
+tags:
+  - API
+  - Audio
+  - Experimental
+  - MSE
+  - Media Source Extensions
+  - Method
+  - Reference
+  - SourceBuffer
+  - Video
+  - remove
```

Comment:
Hi @Jack-Zhang-1314, we have just updated the translation guide, the metadata `tags`, `page-type` is now should be removed from l10n documents: https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE

You can also see the issue about it: #7412

```suggestion
```

Reply:
I see, thank you very much